arangod/Cluster/ClusterInfo.cpp

```diff
@@ -5021,7 +5021,7 @@ ClusterInfo::getPlan(uint64_t& index, std::unordered_set<std::string> const& dir
 
   // We should never proceed here, until we have seen an
   // initial agency cache through loadPlan
-  waitForPlan(1);
```

Comment:
We also need to address `waitForCurrent` calls, I found another one with forgotten `.wait()` or `.get()`, it is in line 5054.